.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Build & Tests (server)
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build_test_run:
+    name: Build and test
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
Any reason to use this version from dtolnay instead of the one from actions-rs like we do in the [esp32-wroom-rp project](https://github.com/Jim-Hodapp-Coaching/esp32-wroom-rp/blob/main/.github/workflows/build_and_test.yml#L20)?

Reply:
Yes! The project is [unmaintained](https://github.com/actions-rs/toolchain/issues/216) and a bunch of stuff was outdated. I haven't checked up on it for a while but people generally recommended switching to dtolnay's version as that is actively maintained and does the same thing.

Reply:
Very good to know, thanks for linking to this here.